components/ActionSheet/BUILD

```diff
@@ -31,6 +31,20 @@ mdc_public_objc_library(
     ],
 )
 
+mdc_objc_library(
+    name = "Theming",
+    srcs = native.glob(["src/Theming/*.m"]),
+    hdrs = native.glob(["src/Theming/*.h"]),
+    includes = ["src/Theming"],
+    deps = [
+        ":ActionSheet",
+        ":ColorThemer",
+        ":TypographyThemer",
+        "//components/schemes/Container",
+    ],
+    visibility = ["//visibility:public"],
```

Comment:
It is my understanding that all external BUILD target visibility is public. We handle visibility restrictions through the Podspec externally was my understanding, I may be incorrect though. Pulling in @randallli and @jverkoey to confirm all BUILD targets visibility are public correct?

Reply:
We haven't put much thought into this aspect of Alpha/Beta for public BUILD files yet mainly because we aren't officially supporting bazel for downstream dependencies, so visibility hasn't been considered that deeply for our BUILD targets.

That being said, I do think it would be valuable to make the Alpha/Beta BUILD targets reflect and communicate the desired visibility. I've filed https://github.com/material-components/material-components-ios/issues/5983 to track further discussion on this topic.

Reply:
[By default, build targets have `visibility:private`](https://docs.bazel.build/versions/master/be/common-definitions.html). The [`mdc_public_objc_library` rule overrides this with `visibility:public`](https://github.com/material-components/material-components-ios/blob/develop/material_components_ios.bzl#L29). Since this non-public target was explicitly marked `visibility:public`, I was surprised.  My understanding was that Theming Extensions were currently in Alpha (only limited client use) and should not be used by most clients. However, since these targets are explicitly marked `public` rather than leaving the default or exposing only the testing targets, I read this change as making the ActionSheet Theming Extension as "Ready" and no longer "Alpha".

Since I might have missed a discussion that included both writing the Action Sheet theming extension and making Theming Extensions generally Ready, I commented on this PR when I saw it in my email.